hack/operatorhub/internal/container/container.go

```diff
@@ -328,7 +328,7 @@ func publishImageInProject(c CommonConfig, newTag Tag, imageScanTimeout time.Dur
 func isImageScanned(c CommonConfig, tag string) (image *Image, done bool, err error) {
```

Comment:
We should maybe update the godoc since the term `passed` no longer seems to be relevant? Also should we change the function name, does it make sense to name it `hasBeenGraded(...)`?

Reply:
Makes sense to me. 👍